code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -714,12 +714,6 @@
 		else
 			dat += text("<tr><td>[module.emag]</td><td><A HREF=?src=\ref[src];act=\ref[module.emag]>Activate</A></td></tr>")
 	dat += "</table>"
-/*
```

Comment:
Remove the whole proc.  It's unused remnant for the white browser module picker.